ansible/roles/kubernetes-addons/templates/dns/kube-svc.yaml.j2

```diff
@@ -1,4 +1,4 @@
-# This file should be kept in sync with cluster/gce/coreos/kube-manifests/addons/dns/skydns-svc.yaml
+# This file should be kept in sync with cluster/gce/coreos/kube-manifests/addons/dns/kubedns-svc.yaml
```

Comment:
see this https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns , the name of the yaml source files are with prefix `kubedns`.